### PR TITLE
html form added

### PR DESCRIPTION
**Pull request by tymchyna**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -13,5 +13,178 @@
   <body>
     <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fieldset_form">
```
Comment (agniya-i) [inline src/index.html:17]:
```suggestion
      <fieldset class="fieldset-form">
```
better to use `-` instead `_`

File: src/index.html [hunk 2]
```diff
@@ -13,5 +13,178 @@
...
+        <legend>Personal information</legend>
+        <label class="form_field">
+          Surname:
+          <input 
+          type="text"
+          name="surname"
+          class="field"
+          autocomplete="off"
+          required
```
Comment (agniya-i) [inline src/index.html:26]:
```suggestion
            type="text"
            name="surname"
            class="field"
            autocomplete="off"
            required
```

add 2 spaces before attributes and fix it **everywhere** in the document

File: src/index.html [hunk 3]
```diff
@@ -13,5 +13,178 @@
...
+          >
+        </label>
+
+        <label class="form_field">
+          Name:
+          <input 
+          type="text"
+          name="name"
+          class="field"
```
Comment (agniya-i) [inline src/index.html:35]: remove the class if you don't use it

File: src/index.html [hunk 4]
```diff
@@ -13,5 +13,178 @@
...
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset_form">
+        <legend>Registration</legend>
+        <label class="form_field">
+          E-mail:
+
+          <input type="email"
```
Comment (agniya-i) [inline src/index.html:84]:
```suggestion
          <input 
            type="email"
```

start first attribute also on new line

File: src/index.html [hunk 5]
```diff
@@ -13,5 +13,178 @@
...
+
+      <fieldset class="fieldset_form">
+        <legend>An interesting fact about you!</legend>
+        <label class="form_field">
+          Do you love cats?
+          <input 
+          type="radio"
+          name="cats"
+          >
+          Yes
```
Comment (agniya-i) [inline src/index.html:113]:
```suggestion
         <label>
            <input 
              type="radio"
              name="cats"
            >
            Yes
          </label>
```

wrap input into label

File: src/index.html [hunk 6]
```diff
@@ -13,5 +13,178 @@
...
+
+          <input
```
Comment (agniya-i) [inline src/index.html:115]: also wrap input into label

File: src/index.html [hunk 7]
```diff
@@ -13,5 +13,178 @@
...
+          type="time"
+          name="bedtime"
+          >
+        </label>
+
+        <label class="form_field">
+          What is your favorite brand of cars?
+
+          <select name="car">
+            <option>BMW</option>
```
Comment (agniya-i) [inline src/index.html:145]:
```suggestion
            <option value="bmw">BMW</option>
```
add values for options

File: src/index.html [hunk 8]
```diff
@@ -13,5 +13,178 @@
...
+          rows="2"
+          >
+          </textarea>
+        </label>
+
+        <label class="form_field">
+          Would you recommend us?
+
+          <select name="yes">
+            <option>yes</option>
```
Comment (agniya-i) [inline src/index.html:181]: add values attributes for options

File: src/index.html [hunk 9]
```diff
@@ -13,5 +13,178 @@
...
+          cols="30" 
+          rows="2"
+          >
+          </textarea>
+        </label>
+
+        <label class="form_field">
+          Would you recommend us?
+
+          <select name="yes">
```
Comment (agniya-i) [inline src/index.html:180]:
```suggestion
          <select name="recommendation">
```

**Review by agniya-i — CHANGES_REQUESTED** [review]
also please fix demo link

File: src/index.html (hunk 2 above)
Comment (tymchyna) [reply src/index.html]: fixed

File: src/index.html (hunk 1 above)
Comment (tymchyna) [reply src/index.html]: fixed

File: src/index.html (hunk 3 above)
Comment (tymchyna) [reply src/index.html]: fixed

File: src/index.html (hunk 4 above)
Comment (tymchyna) [reply src/index.html]: fixed

File: src/index.html (hunk 5 above)
Comment (tymchyna) [reply src/index.html]: fixed

File: src/index.html (hunk 6 above)
Comment (tymchyna) [reply src/index.html]: fixed

File: src/index.html (hunk 7 above)
Comment (tymchyna) [reply src/index.html]: fixed

File: src/index.html (hunk 8 above)
Comment (tymchyna) [reply src/index.html]: fixed

File: src/index.html (hunk 9 above)
Comment (tymchyna) [reply src/index.html]: fixed

**Pull request closed without merging**